src/components/CallView/shared/VideoBottomBar.vue

```diff
@@ -247,6 +258,15 @@ export default {
 		},
 	},
 
+	watch: {
+		mouseover(value) {
+			if (!this.isBig) {
+				return
+			}
```

Comment:
theoretically, this can be and should be checked on a VideoVue level (if we ever wanted to apply a freeze on a Stripe/Grid)

Reply:
I wouldn't apply the freeze too much on the call view ? It's not much needed in grid or stripe as much as in speaker view because in the latter, tile will change on each speaking signal coming in chronologically whereas in the other view, the move is only on the first speaking signal of each speaker and they remain in the same position in first page until they mute their mic. 

Also, it can happen that user lays the cursor on the bottom bar of a tile for a while and many changes took place (a lot of participants spoke and order changed). Once they move it away, the view can jump oddly.